dashboard/config/application.rb

```diff
@@ -80,6 +80,7 @@ class Application < Rails::Application
     cache_bust_path = Rails.root.join('.cache_bust')
     ::CACHE_BUST = File.read(cache_bust_path).strip.gsub('.', '_') rescue ''
 
+    config.assets.paths << Rails.root.join('./public/blockly/js')
```

Comment:
Have you looked to see whether this means whether (1) all of blockly/js ends up being downloaded on every page or (2) the parts of blockly/js that are needed by that page are downloaded?

i.e. if i go to a maze level, am i also downloading turtle code?